.jest/moment.js

```diff
@@ -1,5 +1,5 @@
-const moment = require('moment-timezone')
+/*const moment = require('moment-timezone')
 jest.doMock('moment', () => {
   moment.tz.setDefault('Australia/Melbourne')
   return moment
-})
\ No newline at end of file
+})*/
```

Comment:
Can we double check if this is no longer necessary in `jest.config.js`? We can probably delete if not used anymore.

Reply:
Yes that can be removed @yeniatencio

Reply:
@rubyllamadora @dylankelly, file deleted.